client/view/workbook/settings/settings.js

```diff
@@ -0,0 +1,16 @@
+import settings from './settings.html'
+
+export default mediator => ({
+	name: `settings`,
+	//defaultChild: `select-sheet`,
+	route: `settings/:key`,
+	template: settings,
+	async resolve(data, { key }) {
```

Comment:
When this state becomes a child state of the `workbook` state, we can drop this `resolve`.